lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -85,7 +85,7 @@ <h3 class="hd hd-3">${_("Generate Certificates")}</h3>
                     <input type="button" id="disabled-btn-start-generating-certificates" class="is-disabled" aria-disabled="true" value="${_('Generate Certificates')}"/>
                 % else:
                     <p class="under-heading">
-                        ${_("When you are ready to generate certificates for your course, click Generate Certificates. You do not need to do this<br/>if you have set the certificate mode to on-demand generation.")}
+                        ${Text(_("When you are ready to generate certificates for your course, click Generate Certificates. You do not need to do this<br/>if you have set the certificate mode to on-demand generation."))}
```

Comment:
You'll need to interpolate in `HTML(<br/>)` using a `format()` call.  I prefer when at least the format replacement variables are placed on the following line.